cmd/up.go

```diff
@@ -897,3 +914,30 @@ func printDisplayContext(dev *model.Dev) {
 	}
 	fmt.Println()
 }
+
+// createPIDFile creates a PID file to track Up state and existence
+func createPIDFile(ns, dp string) error {
+
+	fileName := fmt.Sprintf("%s-%s.pid", ns, dp)
+	filePath := filepath.Join(config.GetOktetoHome(), fileName)
```

Comment:
I think is better if the file is called `okteto.pid` and you have it inside `config.GetDeploymentHome`, that's the folder that's meant to keep the files specific to one `okteto up`